packages/flutter_migrate/lib/src/compute.dart

```diff
@@ -0,0 +1,1014 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'base/common.dart';
+import 'base/file_system.dart';
+import 'base/logger.dart';
+import 'base/project.dart';
+import 'base/terminal.dart';
+import 'custom_merge.dart';
+import 'environment.dart';
+import 'flutter_project_metadata.dart';
+import 'result.dart';
+import 'utils.dart';
+
+// This defines files and directories that should be skipped regardless
+// of gitignore and config settings. Using `/` as a stand-in for path separator.
+const List<String> _skippedFiles = <String>[
+  'ios/Runner.xcodeproj/project.pbxproj', // Xcode managed configs that may not merge cleanly.
```

Comment:
This is the bulk of the native build on iOS; is there a plan for including it later? Otherwise this tool isn't going to be able to do much for iOS projects.

Reply:
@jmagman In initial versions of this tool, we don't plan on touching this file as it is at high risk of breaking, but if there are better programmatic ways to approach this (perhaps invoking xcode features), please let me know. Also, if there are other dangerous files in the ios dir, we should include it here.

Reply:
I suggested this file be excluded since diffing `pbxproj` is pretty horrible and prone to project corruption, and the user has legitimate reason to update it (bundle identifier, etc).  We already have a migration mechanism in the tool to update this file so we can be more surgical and not rely on diffing (example https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/ios/migrations/project_base_configuration_migration.dart).